piptools/repositories/pypi.py

```diff
@@ -23,6 +23,7 @@
     path_to_url,
     url_to_path,
 )
+from .._compat.pip_compat import PIP_VERSION, RequirementTracker, WheelCache
```

Comment:
```suggestion
from .._compat import PIP_VERSION, RequirementTracker, WheelCache
```

This can be imported from `_compat` and added to the import list above.